cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -103,6 +104,8 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
         #     name="slurmdbd",
         # )
 
+        self._add_outputs()
```

Comment:
Nitpick: could we call it `_add_stack_outputs()`?

Reply:
I followed the convention used for other PCluster stacks